pubsub/README.md

```diff
@@ -90,7 +92,10 @@ the peer won't be able to send identical messages (e.g. keepalives) within the
 timecache interval, as they will get rejected as duplicates."
 
 The `topicIDs` field specifies a set of topics that this message is being
-published to.
+published to. To prevent the size of this field from growing indefinitely,
+it is suggested that that the application using an implementation of this
+library spec should have some kind of storage economics (see e.g.
+[here](https://ethresear.ch/t/draft-position-paper-on-resource-pricing/2838).
```

Comment:
that's probably too much information, is it even relevant to recommend storage economics?

Reply:
Maybe not so much for non-blockchain applications.